elements/pfe-toast/src/pfe-toast.scss

```diff
@@ -1,32 +1,42 @@
 @import "../../pfe-sass/pfe-sass";
 $LOCAL: toast;
 
-:host {
-  // local variables
-  --pfe-toast--Top: 50px;
-  --pfe-toast--MaxWidth: 500px;
-  --pfe-toast--MinWidth: 250px;
-  --pfe-toast--Right--offset: 50px; // needed with box-shadow
-  --pfe-toast--Right: calc(-1 * (#{pfe-local(MaxWidth)} + #{pfe-local(Right--offset)}));
+$LOCAL-VARIABLES: (
+  Top: 50px,
+  MaxWidth: 500px,
+  MinWidth: 250px,
+  Right--offset: 50px, // needed with box-shadow
 
-  --pfe-toast__container--BackgroundColor: #{pfe-color(surface--lightest)};
-  // --pfe-toast__container--BoxShadow: #{pfe-var(box-shadow--lg)};
-  // @TODO update this to use pfe-sass box-shadow variables
-  --pfe-toast__container--BoxShadow: rgba(3, 3, 3, 0.13) 0px 3px 7px 3px, rgba(3, 3, 3, 0.12) 0px 11px 24px 16px;
-  --pfe-toast__container--Color: #{pfe-color(text)};
-  --pfe-toast__container--Padding: 1rem;
+  container: (
+    BackgroundColor: pfe-var(surface--lightest),
+    // BoxShadow: #{pfe-var(box-shadow--lg)},
+    // @TODO update this to use pfe-sass box-shadow variables
+    BoxShadow: #{rgba(3, 3, 3, 0.13) 0px 3px 7px 3px, rgba(3, 3, 3, 0.12) 0px 11px 24px 16px},
+    Color: pfe-var(text),
+    Padding: 1rem
+  ),
 
-  --pfe-toast__close--svg--Fill:        #{pfe-color(feedback--default--darkest)};
-  --pfe-toast__close--svg--Fill--hover: #{$pfelements--gray-dark};
-  --pfe-toast__close--svg--Height:   12px;
-  --pfe-toast__close--svg--Width:    12px;
-  
-  --pfe-toast__close--PaddingTop:    0;
-  --pfe-toast__close--PaddingRight:  0;
-  --pfe-toast__close--PaddingBottom: 0;
-  --pfe-toast__close--PaddingLeft:   0;
-  --pfe-toast__close--Padding:       #{pfe-local($cssvar: PaddingTop, $region: close)} #{pfe-local($cssvar: PaddingRight, $region: close)} #{pfe-local($cssvar: PaddingBottom, $region: close)} #{pfe-local($cssvar: PaddingLeft, $region: close)};
+  close: (
+    svg--Fill: pfe-var(feedback--default--darkest),
+  	svg--Fill--hover: $pfelements--gray-dark,
+  	svg--Height: 12px,
+  	svg--Width: 12px,
   
+  	PaddingTop: 0,
+  	PaddingRight: 0,
+  	PaddingBottom: 0,
+  	PaddingLeft: 0
+  )
+);
+
+$LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, (
```

Comment:
@starryeyez024 This is a good example of why some values have to exist in a 2nd map because they are self-referencing

Reply:
👍